kaiax/gasless/impl/init.go

```diff
@@ -30,13 +31,14 @@ var logger = log.NewModuleLogger(log.KaiaxGasless)
 type InitOpts struct {
 	ChainConfig *params.ChainConfig
 	NodeKey     *ecdsa.PrivateKey
+	TxPool      kaiax.TxPoolForCaller
```

Comment:
It seems unused

Reply:
It's being used for getting the current state db.